src/core_plugins/kibana/public/dashboard/top_nav/add_panel.html

```diff
@@ -4,24 +4,26 @@
 >
   <div class="kuiLocalDropdownTitle">Add Panels</div>
 
-  <ul class="nav nav-tabs">
-    <li ng-class="{active: mode == 'visualization'}">
-      <a
-        ng-click="mode='visualization'"
-        aria-label="List visualizations"
-      >
-        Visualization
-      </a>
-    </li>
-    <li ng-class="{active: mode == 'search'}">
-      <a
-        ng-click="mode='search'"
-        aria-label="List saved searches"
-      >
-        Saved Search
-      </a>
-    </li>
-  </ul>
+  <div class="kuiTabs">
+    <div
```

Comment:
These can be `<button>` elements, and then we won't need `kbn-accessible-click` any more. That directive is meant to be used as a last resort, if you can't use a `<button>` or `<a>` with an `href`.